os/Makefile.unix

```diff
@@ -485,6 +485,7 @@ define VERIFY_APP
 		rm $(OUTBIN_DIR)/$1; \
 		exit 1; \
 	fi
+	$(Q) $(TOPDIR)/tools/mkverifyappsize.py $(OUTBIN_DIR)/$1 $2
```

Comment:
This must be done only in case of chipsets which support MMU. Please check for mmu config here.

Reply:
I have added if statement for config check.